distribution/scripts/getbuildvmtk.py

```diff
@@ -22,7 +22,7 @@
 VTK_BUILD_DIR = os.path.join(WORK_DIR, 'vtk-build')
 ITK_BUILD_DIR = os.path.join(WORK_DIR, 'itk-build')
 VMTK_BUILD_DIR = os.path.join(WORK_DIR, 'vmtk-build')
-PARALLEL_JOBS = 3
+PARALLEL_JOBS = 7
```

Comment:
7 sounds too much (I'm thinking about my laptop :-) ). I'd stick with 3 by default and eventually add a command-line flag to increase it.